lib/monkey/include/monkey/mk_core/mk_event.h

```diff
@@ -52,6 +52,9 @@
 #define MK_EVENT_NONE            1    /* nothing */
 #define MK_EVENT_REGISTERED      2    /* event is registered into the ev loop */
 
+/* Priority bucket queue */
+#define MK_EVENT_PRIORITY_DEFAULT 6   /* default priority */
```

Comment:
Is this a good default priority? I figured we would need to prioritize events more than deprioritize events, so 6 prioritized event priorities and 1 deprioritized event priorities. (8 priorities total)

5 or even 4 might be a better option, though I thought 6 was the best at the time.